app/validators/edtf_validator.rb

```diff
@@ -3,7 +3,7 @@
 
 class EdtfValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    return unless EDTF.parse(value).nil?
+    return if value.nil? || EDTF.parse(value)
```

Comment:
This change allows `nil` `created_edtf` values? Can you say more about that?

Reply:
No, this means "don't validate if they don't have a date".  If we want it to be not nil, that should be done by the presence validator.

Reply:
No longer needs commenting.